graphx/src/main/scala/org/apache/spark/graphx/GraphOps.scala

```diff
@@ -424,6 +424,18 @@ class GraphOps[VD: ClassTag, ED: ClassTag](graph: Graph[VD, ED]) extends Seriali
     PageRank.run(graph, numIter, resetProb)
   }
 
+  /**
+   * Run PageRank for a fixed number of iterations returning a graph with vertex attributes
+   * containing the PageRank and edge attributes the normalized edge weight, optionally including
+   * including a previous pageRank computation to be used as a start point for the new iterations
+   *
+   * @see [[org.apache.spark.graphx.lib.PageRank$#runWithOptionsWithPreviousPageRank]]
+   */
+  def staticPageRank(numIter: Int, resetProb: Double,
```

Comment:
Why do we need this in GraphOps?

Reply:
Well, to have another version of staticPageRank, it seem natural to have in the same place as the function it overloads, also it needs to be visible from Graph.

Reply:
Oops right, disregard. I lost track.